extras/postprocess_split_xdf.bat

```diff
@@ -0,0 +1,2 @@
+call  C:\Users\CTR\anaconda3\Scripts\activate.bat C:\Users\CTR\anaconda3\envs\neurobooth
+call start /W ipython C:\neurobooth-eel\extras\run_xdf_split_postproces.py
```

Comment:
Should this be 'neurobooth-os' instead of neurobooth-eel ??

Reply:
at the neurobooth it's still neurobooth-eel. For now, change it to neurobooth-os for staging